network-api/networkapi/buyersguide/pagemodels/cloudinary_image_field.py

```diff
@@ -2,7 +2,7 @@
 
 
 # Override the default 'public_id' to upload all images to the buyers guide directory on Cloudinary
-# TODO: this is not needed anymore and needs to be removed.
+# TODO: this is not needed anymore and needs to be removed when we remove the rest of Cloudinary
```

Comment:
This can be removed when migrations are squashed which I believe is in progress at the moment.